ext/LogDensityProblemsADDifferentiationInterfaceExt.jl

```diff
@@ -0,0 +1,53 @@
+module LogDensityProblemsADDifferentiationInterfaceExt
+
+if isdefined(Base, :get_extension)
+    import LogDensityProblemsAD
+    import ADTypes
+    import DifferentiationInterface as DI
+else
+    import ..LogDensityProblemsAD
+    import ..ADTypes
+    import ..DifferentiationInterface as DI
+end
```

Comment:
The package only supports Julia >= 1.10:
```suggestion
import LogDensityProblemsAD
import ADTypes
import DifferentiationInterface as DI
```